Geometry/TrackerGeometryBuilder/interface/TrackerGeometry.h

```diff
@@ -97,7 +105,7 @@ class TrackerGeometry final : public TrackingGeometry {
 
   GeomDetEnumerators::SubDetector theSubDetTypeMap[6];
   unsigned int theNumberOfLayers[6];
-
+  std::vector< std::tuple< DetId, TrackerGeometry::ModuleType, float> > theDetTypetList; 
```

Comment:
@suchandradutta - is it filled for all DetIds? Does it change from run to run or only from era to era? Perhaps, a persistent record (DB) associated with GT would be better?